app/main.ipynb

```diff
@@ -126,19 +163,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
-    "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
-    "# write your code here"
+    "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1], dtype=np.float16)\n",
+    "array[[0, 1, 4, -1]].sum()"
```

Comment:
The selection of elements and summation to achieve the result of 27 is correctly implemented using multi-indexing.